apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -65,38 +82,93 @@ AprilTagDetector::AprilTagDetector(ros::NodeHandle& nh, ros::NodeHandle& pnh): i
 
   pnh.param<bool>("start_enabled", enabled_, false);
 
+  pnh.param<bool>("publish_plane_cloud", publish_plane_cloud_, false);
+
+  pnh.param<float>("plane_inlier_threshold", plane_inlier_threshold_, 0.7f);
+  plane_inlier_threshold_ = std::max(0.0f, std::min(1.0f, plane_inlier_threshold_));
+
+  pnh.param<float>("plane_angle_threshold", plane_angle_threshold_, 5.0f);
+  plane_angle_threshold_ = std::max(0.0f, std::min(90.0f, plane_angle_threshold_));
+
+  // Read parameters
+  int queue_size = 5;
+  pnh.param("queue_size", queue_size, 5);
+
+  ROS_INFO("April tag info: start_enabled: %d, publish_plane_cloud: %d, plane_inlier_threshold: %f",
+    enabled_,
+    publish_plane_cloud_,
+    plane_inlier_threshold_);
+
   enable_sub_ = pnh.subscribe("enable", 1, &AprilTagDetector::enableCb, this);
   tag_detector_= boost::shared_ptr<AprilTags::TagDetector>(new AprilTags::TagDetector(*tag_codes));
-  image_sub_ = it_.subscribeCamera("image_rect", 1, &AprilTagDetector::imageCb, this);
+
+  rgb_it_.reset( new image_transport::ImageTransport(nh) );
+
+  sync_.reset( new Synchronizer(SyncPolicy(queue_size), sub_point_cloud_, sub_rgb_, sub_info_) );
+  sync_->registerCallback(boost::bind(&AprilTagDetector::imageCb, this, _1, _2, _3));
+
+  sub_rgb_.subscribe(*rgb_it_, "image_rect", 5);
```

Comment:
These all have the same queue size as "queue_size".  Should they use that variable or be set to 5 all the time?